provider/java/dependency.go

```diff
@@ -37,8 +38,24 @@ func (p *javaProvider) GetDependencies() ([]dependency.Dep, error) {
 	return deps, err
 }
 
+func (p *javaProvider) getLocalRepoPath() string {
+	cmd := exec.Command("mvn", "help:evaluate", "-Dexpression=settings.localRepository", "-q", "-DforceStdout")
+	var outb bytes.Buffer
+	cmd.Stdout = &outb
+	err := cmd.Run()
+	if err != nil {
+		fmt.Printf("here: %v", err)
```

Comment:
~this guy too :)~

Actually I think we need this error reported right?

Reply:
yeah, it probably should be a log but this is a good first issue we have these all over the place